driver/csiplugin/gpfs.go

```diff
@@ -222,10 +225,14 @@ func (driver *ScaleDriver) SetupScaleDriver(ctx context.Context, name, vendorVer
 	}
 	_ = driver.AddControllerServiceCapabilities(ctx, csc)
 
-	ns := []csi.NodeServiceCapability_RPC_Type{
-		csi.NodeServiceCapability_RPC_GET_VOLUME_STATS,
+	statsCapability := os.Getenv(volumeStatsCapability)
+	if strings.ToUpper(statsCapability) != "DISABLED" {
+		klog.Infof("[%s] volume stats capabililty is enabled", utils.GetLoggerId(ctx))
+		ns := []csi.NodeServiceCapability_RPC_Type{
+			csi.NodeServiceCapability_RPC_GET_VOLUME_STATS,
+		}
+		_ = driver.AddNodeServiceCapabilities(ctx, ns)
```

Comment:
I think we may need to return the empty list of capacities in case on else case. Please check. This is something we used to do before stat was added. 
	ns := []csi.NodeServiceCapability_RPC_Type{}
	_ = driver.AddNodeServiceCapabilities(ns)

Reply:
Done